### ISSUE TITLE
Tecplot

### ISSUE THREAD

**Issue by laraPPr** [vscentrum/vsc-software-stack#350]
* link to support ticket: [#2024060460000806](https://otrsdict.ugent.be/otrs/index.pl?Action=AgentTicketZoom;TicketID=158633)
* website: https://tecplot.com/
* installation docs: https://tecplot.azureedge.net/products/360/current/360_install_guide.pdf
* toolchain: `/`
* easyblock to use: `Binary`
* required dependencies:
  * [ ] ...
* optional dependencies:
  * [ ] ...
* notes:
  * Already have easyconfig https://github.com/easybuilders/easybuild-easyconfigs/blob/0c80144b30fd75b4729dad480e09efbf2505c507/easybuild/easyconfigs/t/tecplot360ex/tecplot360ex-linux64.eb#L6
  * Need license server on Hortense
* effort: *(TBD)*
* other install methods
  * conda: yes (link?) / no
  * container image: yes (link?) / no
  * pre-built binaries (RHEL8 Linux x86_64): yes (link?) / no
  * easyconfig outside EasyBuild: yes / no

**Comment by boegel** [comment]
See existing easyconfig `tecplot360ex-linux64.eb`?

**Comment by laraPPr** [comment]
yes I opened it because I did not know what to do for the licence

**Comment by laraPPr** [comment]
This has gone stale. I have closed the ticket. So will close this ticket until we hear back from the user.
